socialhome/streams/app/stores/streamStore.js

```diff
@@ -1,31 +1,52 @@
 import Vue from "vue"
 import Vuex from "vuex"
-import isNil from "lodash/isNil"
+import ReconnectingWebSocket from "reconnecting-websocket"
 
-// Vue setup
+import getState from "streams/app/stores/streamStore.state"
+import {actions, mutations, stateOperations} from "streams/app/stores/streamStore.operations"
+
+/* This should actually rely in ``main.js``.
+ *
+ * However, a bug in Webpack, or Babel loader, it's not clear,
+ * Make imports statements always be on top of the generated
+ * JS. This means that, even if imports are done this way
+ * in ``main.js``:
+ *
+ *     import Vue from "vue"
+ *     import Vuex from "vuex"
+ *
+ *     Vue.use(Vuex)
+ *
+ *     import "streams/app/components/streamStore.vue"
+ *
+ * The ``import "streams/app/components/streamStore.vue"``
+ * statetment will always be generated before ``Vue.use(Vuex)``,
+ * making Vue to raise an error on client-side saying that
+ * ``[vuex] must call Vue.use(Vuex) before creating a store instance``
+ *
+ * See https://github.com/webpack/webpack/issues/1343 for
+ * more details.
+ */
 Vue.use(Vuex)
 
-function getContext() {
-    try {
-        return {
-            state: {
-                translations: {
-                    stampedContent: {
-                        h2: window.context.translations.stampedContent.h2,
-                        p: window.context.translations.stampedContent.p,
-                    },
-                },
-                contentList: window.context.contentList,
-                streamName: window.context.streamName,
-                isUserAuthenticated: window.context.isUserAuthenticated,
-                showAuthorBar: window.context.showAuthorBar,
-                currentBrowsingProfileId: (isNil(window.context.currentBrowsingProfileId)
-                    ? undefined : `${window.context.currentBrowsingProfileId}`),
-            },
+function newinstance() {
+    const state = getState()
+    const store = new Vuex.Store({state, mutations, actions})
+    const wsProtocol = window.location.protocol === "https:" ? "wss" : "ws"
+    const wsPath = `${wsProtocol}://${window.location.host}/ch/streams/${state.stream.streamName}/`
+    const ws = new ReconnectingWebSocket(wsPath)
+
+    ws.onmessage = message => {
+        const data = JSON.parse(message)
+
+        if (data.event === "content") {
+            store.dispatch(stateOperations.receivedNewContent, data.contents.length)
```

Comment:
I think the message doesn't contain a `contents`. Did you print it out?

See https://github.com/jaywink/socialhome/blob/67c12ee943c12125c414462c7052d28230e7ffce/socialhome/content/signals.py#L58